README.md

```diff
@@ -12,11 +12,9 @@ This script downloads any Git repository and extracts statistics from its commit
 To use it, run the following command:
 
 ```bash
-npx @jointly/spyone <repo-url> <days-to-cover> <branch-name> <output-format>
+npx @jointly/spyone
```

Comment:
This would remain as before but I'd add a paragraph saying if you don't pass any parameter it will go in interactive mode.